Lib/test/test_eagerly_import.py

```diff
@@ -0,0 +1,82 @@
+'''
+- Target:
+    Add a new sample test that checks Python imports are executed eagerly by default.
+
+- Description:
+    We need to confirm one of these two options based on the scenario. Otherwise, this test fails.
+        - If it is in default, the imports are executed eagerly.
+        - If Lazy Imports is enabled, the imports are NOT executed eagerly.
+'''
+
+import os
+import sys
+import unittest
+# import json
+import sqlite3
+
+class LazyImportsEnabledChecker:
+    def __init__(self):
+        self.enabled = self.check_lazy_imports()
+
+    # confirm if Lazy Imports is enabled or not
+    def check_lazy_imports(self):
+        # lazy imports is not enabled in default
+        enabled = False
+
+        # check if Lazy Imports is turned on
+        enabled |= self.check_flag()
+        enabled |= self.check_env_var()
```

Comment:
It's probably not as relevant in this particular diff because these functions as not as expensive, but to avoid calling these methods once one has succeeded, you could have short-circuited like this:

```py
    return (
        self.check_flag() |
        self.check_env_var()
    )
```

This way `self.check_env_var()` doesn't get called if the first one returned `True`.